docker-entrypoint.sh

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
-# setup container user
-echo "viya4-deployment:*:$(id -u):$(id -g):,,,:/viya4-deployment:/bin/bash" >> /etc/passwd
-echo "viya4-deployment:*:$(id -G | cut -d' ' -f 2)" >> /etc/group
+echo "viya4-deployment:x:$(id -u):$(id -g):Viya4:/viya4-deployment:/bin/bash" >> /etc/passwd
+echo "viya4-deployment:x:$(id -G | cut -d' ' -f 1):" >> /etc/group
```

Comment:
This cannot be changed. What is the need here to alter what's currently in place?

Reply:
The change is mostly cosmetic as the proper format for those files in Linux OS is in the example below
﻿[sshulgin@fedora viya4-deployment]$ uname -a
Linux fedora 5.15.10-100.fc34.x86_64 #1 SMP Fri Dec 17 14:51:10 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
[sshulgin@fedora viya4-deployment]$ grep sshulgin /etc/passwd
sshulgin:x:1000:1000:Serg:/home/sshulgin:/bin/bash
[sshulgin@fedora viya4-deployment]$ grep sshulgin /etc/group
wheel:x:10:sshulgin
sshulgin:x:1000:

Reply:
The format of the file is correct in the user description field you can have several items separated by a , You can shorten the field but the changes should not be necessary for Fedora or any Linux variant. Are you seeing issues with the current format or is there a bug here?

Reply:
So that's a good catch there. This is probably a bug for the command. Will look at changing that. The :Z issue still needs to be investigated.